packages/react-native/src/primitives/TextField/TextField.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useMemo } from 'react';
+import { TextInput, View, ViewStyle } from 'react-native';
+import { Label } from '../Label';
+
+import { styles } from './styles';
+import { TextFieldProps } from './types';
+
+export default function TextField({
+  accessibilityLabel,
+  containerStyle,
+  disabled,
+  error,
+  errorMessage,
+  errorMessageStyle,
+  label,
+  labelStyle,
+  password,
+  textStyle,
+  type = 'default',
+  ...rest
+}: TextFieldProps): JSX.Element {
+  const inputContainerStyle: ViewStyle = useMemo(
+    () => ({
+      ...styles.container,
+      ...(disabled && styles.disabled),
+    }),
+    [disabled]
+  );
+
+  return (
+    <View style={[inputContainerStyle, containerStyle]}>
+      {label ? <Label style={labelStyle}>{label}</Label> : null}
+      <TextInput
+        accessible
```

Comment:
Should this be configurable by users?

Reply:
Upon further inspection of APIs, this is actually not needed, all components are accessible by default. Whether we allow configuration can be discussed as follow-up.